django/forms/boundfield.py

```diff
@@ -286,6 +286,11 @@ def widget_type(self):
             r"widget$|input$", "", self.field.widget.__class__.__name__.lower()
         )
 
+    @property
+    def use_fieldset(self):
+        """Returns the value of this BoundField widget's use_fieldset attribute."""
```

Comment:
We still wrap comments and docstrings at 79 chars. Also `Return` instead of `Returns` per PEP-257:
```suggestion
        """
        Return the value of this BoundField widget's use_fieldset attribute.
        """
```